db.yml

```diff
@@ -2287,11 +2287,43 @@ modules:
           sceGxmVertexFence: 0xE05277D6
           sceGxmVertexProgramGetProgram: 0xBC52320E
           sceGxmWaitEvent: 0x8BD94593
+      SceGxmInternal:
+        kernel: false
+        nid: 0x3FE654E6
+        functions:
+          sceGxmCheckMemoryInternal: 0x88B424FB
+          sceGxmCreateRenderTargetInternal: 0xC8A0F04E
+          sceGxmGetDisplayQueueThreadIdInternal: 0xAB1D9466
+          sceGxmGetRenderTargetMemSizeInternal: 0x4DD98588
+          sceGxmGetTopContextInternal: 0x90EE6A28
+          sceGxmInitializedInternal: 0xC8E9F108
```

Comment:
I guess this and sceGxmIsInitialization are erroneously swapped?

Reply:
One returns a bool value and the other returns 0 or an error code.